grammars/css.cson

```diff
@@ -20,294 +20,1171 @@
+          (?:min-|max-)?            # Range features
+          (?: height
+            | width
+            | aspect-ratio
+            | color
+            | color-index
+            | monochrome
+            | resolution
+          )
+          | grid                    # Discrete features
+          | scan
+          | orientation
+          | display-mode
+        )
+        |
+        # Deprecated features
+        (
+          (?:min-|max-)?            # Deprecated in Media Queries 4
+          device-
+          (?: height
+            | width
+            | aspect-ratio
+          )
+        )
+        |
+        # Vendor extensions
+        (
+          (?:
+            # Spec-compliant syntax
+            [-_]
+            (?: webkit              # Webkit/Blink
+              | apple|khtml         # Webkit aliases
+              | epub                # ePub3
+              | moz                 # Gecko
+              | ms                  # Microsoft
+              | o                   # Presto (pre-Opera 15)
+              | xv|ah|rim|atsc|     # Less common vendors
+                hp|tc|wap|ro
+            )
+            |
+            # Non-standard prefixes
+            (?: mso                 # Microsoft Office
+              | prince              # YesLogic
+            )
+          )
+          -
+          [\\w-]+                   # Feature name
+          (?=                       # Terminates correctly
+            \\s*                    # Possible whitespace
+            (?:                     # Possible injected comment
+              /\\*
+              (?:[^*]|\\*[^/])*
+              \\*/
+            )?
+            \\s*
+            [:)]                    # Ends with a colon or closed bracket
+          )
+        )
+      )
+      (?=\\s|$|[><:=]|\\)|/\\*)     # Terminates cleanly
+    '''
+  'media-feature-keywords':
+    'match': '''(?xi)
+      (?<=^|\\s|:|\\*/)
+
+      (?: portrait                  # Orientation
+        | landscape
+        | progressive               # Scan types
+        | interlace
+        | fullscreen                # Display modes
+        | standalone
+        | minimal-ui
+        | browser
+      )
+
+      (?=\\s|\\)|$)
+    '''
+    'name': 'support.constant.property-value.css'
+  'media-query':
+    'begin': '\\G'
+    'end': '\\s*(?=[{;])'
     'patterns': [
       {
-        'match': '\\s*(\\$[A-Za-z0-9_-]+\\b)\\s*'
-        'name': 'variable.css'
-      }
-      {
-        'include': '#property-values'
+        'include': '#comment-block'
       }
       {
-        'include': '#string-double'
+        'include': '#escapes'
       }
       {
-        'include': '#string-single'
+        'include': '#media-types'
       }
-    ]
-  }
-  {
-    'begin': '(?=\\{)'
-    'end': '(?!\\G)'
-    'patterns': [
       {
-        'include': '#rule-list'
+        'match': '(?i)(?<=\\s|^|,|\\*/)(only|not)(?=\\s|$|{|/\\*)'
+        'name': 'keyword.operator.logical.$1.media.css'
```

Comment:
(Like similar questions before) can this and the one right after this simply be `keyword.operator.logical.media.css`?